public/styles.css

```diff
@@ -1576,3 +1576,235 @@ nav a {
     margin-left: 40%;
   }
 }
+
+/* The following css is for the users page */
```

Comment:
NIce work! But instead of putting all css here. Each page can have their own dedicate css files.